app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/TrackListViewModel.kt

```diff
@@ -154,6 +154,7 @@ constructor(
 
   data class UiState(
       val tracks: List<Track> = listOf(),
+      val queue: List<Track> = listOf(),
```

Comment:
idk if you know it, but there is a queue data structure in kotlin if that might be usefull for you

Reply:
Thanks for letting me know, but in this case a list is just what I need